tpm2/tpm2.go

```diff
@@ -513,6 +513,14 @@ func CreateKey(rw io.ReadWriter, owner tpmutil.Handle, sel PCRSelection, parentP
 	return private, public, nil
 }
 
+// CreateKeyEx creates a new key pair under the owner handle.
+// Returns private key and public key blobs.
+// This function differs from CreateKey in that all response elements
+// are returned.
+func CreateKeyEx(rw io.ReadWriter, owner tpmutil.Handle, sel PCRSelection, parentPassword, ownerPassword string, pub Public) ([]byte, []byte, error) {
```

Comment:
The "Ex" suffix is a Windows idiom. That doesn't make it bad or good, but I don't think this API is currently using that naming scheme.

Does this compile? It looks to me like the return type of CreateKeyEx is []byte, []byte, error, but the return type of the create function is much longer.

Reply:
Yes, that was my mistake, forgot to call git add before doing commit --amend

Reply:
as for Ex, tpm2 seems to use it already for CreatePrimaryEx

Reply:
We were using `*Ex` pattern to preserve backwards-compatibility.
Now that this repo is a Go module, we can break compatibility and tag new versions.

Please change `CreatePrimary` instead and name all the return params. All those byte slices are a bit obscure.